README.md

```diff
@@ -50,7 +50,15 @@ an asynchronous application.
 
 ## Example
 
-A basic TCP echo server with Tokio:
+A basic TCP echo server with Tokio.
+
+Make sure you activated the full features of the tokio crate on Cargo.toml:
+
+```
```

Comment:
The test setup is trying to run this as Rust code. The following change disables that.
```suggestion
```text
```